releasenotes/notes/dogpile.cache-1d03e088872ca747.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    dogpile.cache: handle both kwargs and args in the wrapper functions (using
+      only kwargs would result in an IndexError.
```

Comment:
```suggestion
    ``dogpile.cache``: handle both ``kwargs`` and ``args`` in the wrapper functions (using
      only ``kwargs`` would result in an ``IndexError``.
```

This is an alternative way to avoid adding to the spelling wordlist.